data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1877 @@ mission "Stone of our Fathers 5"
+			scene "scene/iceplains2"
+			choice
+				`	(You look out at the bright Ice-Plains as they slowly disappear into the dull horizon.)`
+					goto rock
+			label noquestion
+			`	"Is smart, not to ask question. Hjlod tell you best parts only." Hjlod gives you a quick grin and swivels back into her makeshift cockpit.`
+			`	"Enough ice-plain, now ve go to next stop."`
+			scene "scene/iceplains2"
+			choice
+				`	(You look out at the bright Ice-Plains as they slowly disappear into the dull horizon.)`
+					goto rock
+			label rock
+			`	The Ice-Crawler plows through snow, hill, mountain, and tundra for another hour or two before a massive and oddly shaped rock passes into view. Hjlod stops the Ice-Crawler so you can look at it.`
+			`	"This is rock." She says matter-of-factly.`
+			`	You look at it for a while and realize there is a face half chiseled into it.`
+			`	"You are done looking? Ve are good to move to next stop?" Hjlod asks.`
+			scene "scene/lonelyrock"
+			choice
+				`	"Yep, I don't need to look at this rock any longer."`
+					goto byerock
+				`	"Wait, is that part of a man's face carved into it? What is so special about this rock?"`
+					goto manrock
+			label manrock
+			`	"That rock is supposed to celebrate man who discover Rigellian cure. He was called Hank, or Henry, or Bill, or something." Hjlod shrugs with evident disdain.`
+			`	"The cure was found here, but back when tundra was melting and Nifel was a little varmer. They say this man discover it and they want to make grand statue for him. But it turn out Nifel not so great place for statue. Beside, no one like statue of liar."`
+			`	Hjlod turns back to her controls and gets the Ice-Crawler moving forward again. "That is enough about stupid rock and stupider man."`
+			`	"Now ve go to river."`
+					goto riverlike
+			label byerock
+			`	"Now ve go to river."`
+					goto riverlike
+			label riverlike
+			choice
+				`	(The large oddly shaped rock falls out of your field of view as the Ice-Crawler turns away from it.)`
+			scene "scene/snowfield"
+					goto river
+			label river
+			`	The uneventful journey to the boiling ice river of Slidr is suddenly interrupted when the Ice-Crawler begins to shake wildly for a few moments, followed by a thunderous boom from somewhere outside.`
+			`	After Hjlod regains control of the crawler, she immediately veers away from the noise. "That vas Nifel-quake, and loud noise was avalanche. This is not good."`
+			`	Despite her calm demeanor she wields the Ice-Crawler with reckless abandon as she tries to avoid the rumbling snow. The machine protests with creaks and whirrs and pops as it goes faster than it has gone for hundreds of years.`
+			`	"I think..." Hjlod starts to say when a mountain of snow crashes into the Ice-Crawler dragging it half a kilometer.`
+			`	You and Hjlod hold tight to your seat-belts as you're tossed around. Eventually, the crawler comes to a rest on its' side.`
```

Comment:
I would say it is _ok_ now, but could be improved further. Treat this as a range of suggestions that you can pick and choose from as it suits you. If you choose to reject this I won't shout about it. :P

```suggestion
			`	The uneventful journey to the boiling ice river of Slidr is suddenly interrupted when a deep tremor followed by a thunderous boom shakes the Ice-Crawler to its core and causes it to slip on the disturbed ice and snow.   Somewhere outside subsequent rumble starts to build rapidly.`
			`	After Hjlod regains control of the crawler, she immediately veers away from the noise. "That vas Nifel-quake, and loud noise was avalanche, or landslide, maybe both. This is not good."`
			`	Despite her calm demeanor she wields the Ice-Crawler with reckless abandon as she tries to avoid the rumbling snow. The machine protests with creaks and whirrs and pops as it strains to claw itself out of the way faster than it has probably gone for hundreds of years. The closer the rumbling gets, the more difficulty Hjlod seems to have retaining traction.`
			`	"I think..." Hjlod starts to say when a mountain of snow crashes into the Ice-Crawler with deafening force. All sense of travelling distance is lost as the snow blots out the windows and the crawler is dragged bumping and twisting over uneven terrain.`
			`	You and Hjlod hold tight to your seat-belts as you're tossed around until, eventually, the crawler grinds up against something reasonably solid and comes suddenly to a rest, partially on its' side.`
			`	A few moments later the rumbling finishes passing you by.`
```